.github/workflows/release-drafter.yml

```
@@ -47,7 +48,31 @@ jobs:
+          script: |
+            const { OpenAI } = require('openai')
+            const openai = new OpenAI({ apiKey: "${{ secrets.OPENAI_API_KEY }}"})
+            const response = await openai.chat.completions.create({
+              model: "gpt-4o-mini",
+              messages: [
+                { role: 'system', content: `You are a slack message writer. Your job is to translate a changelog to a short message (1 or 2 sentences) about what is changed. This message will be read by colleagues in our company. You can make use of 'mrkdwn' to highlight keywords or the make the message prettier. When a changelog is provided, respond with the message.` },
+                { role: 'user', content: `${{ steps.release-drafter.outputs.body }}` }
+              ]
+            })
+            console.log("::set-output name=generated_text::" + response.choices[0].message.content)
+      
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)